CMSgov/onemac
# Oy2 19021 - Medicaid SPA RAI Response

Story: https://qmacbis.atlassian.net/browse/OY2-19021
Endpoint: https://d1nl90l70m75c.cloudfront.net/

### Details

Added ability to submit medicaid SPA RAI forms.

### Changes

- added/updated form and configuration for medicaid spa rai
- created componentId component to replace TransmittalId component on OneMacForm
- added logic to submit for forms marked with allowMultiplesWithSameId=false

### Test Plan

1. Login as state submitter
2. Locate a Medicaid SPA in status of "RAI Issued" and navigate to the details page
3. Click the "Respond to RAI" action link
4. Verify the SPA ID populates with the SPA ID you created the response from
5. Add attachments and optional text and submit
6. Verify you are redirected back to the Medicaid SPA details page
7. Verify the submitted RAI Response appears in the RAI Responses section of the details page
